client/public/index.html

```diff
@@ -20,22 +20,28 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-  <base href="/">
-  <!-- CSS  -->
-  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-  <link href="materializeCSS/materialize.css" type="text/css" rel="stylesheet" media="screen,projection" />
-  <link href="materializeCSS/materialize_style.css" type="text/css" rel="stylesheet" media="screen,projection" />
-  <!--  Scripts-->
-  <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
-  <script src="materializeJS/materialize.js"></script>
-  <script src="materializeJS/init.js"></script>
-  <title>Piggy Business React Page</title>
+    <base href="/">
+    <!-- CSS  -->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Amatic+SC">
+    <link href="materializeCSS/materialize.css" type="text/css" rel="stylesheet" media="screen,projection" />
+    <link href="materializeCSS/materialize_style.css" type="text/css" rel="stylesheet" media="screen,projection" />
+    <!--  Scripts-->
+    <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
+    <script src="materializeJS/materialize.js"></script>
+    <script src="materializeJS/init.js"></script>
+    <title>Piggy Business React Page</title>
 </head>
 
-<body>
+<body style="font-family: 'Amatic SC', cursive">
   <noscript>You need to enable JavaScript to run this app.</noscript>
-  <div id="root"></div>
+  <div id="root" style="height: 100%">
 
+  </div>
+  <script>
+  $(document).ready(function(){
```

Comment:
again with jquery. We could have had everything in react, but y'know if it's working it's working. After this project i do highly recommend getting this out and sticking to react and figuring out how to create a modal using a prebuilt component or your own custom modal in react.

Reply:
Skipping this since we don't have the time to complete.